src/Forms/TreeDropdownField.php

```diff
@@ -633,6 +598,9 @@ public function getAttributes()
     }
 
     /**
+     * HTML-encoded label for this node, including css classes and other markup.
+     *
+     * @deprecated 4.0...5.0 Use setTitleField()
```

Comment:
would a deprecation notice for these two methods be worth it?

Reply:
We still need to actually call this method, so that would trigger errors even in code that doesn't call it.